shared/js/background/events.es6.js

```diff
@@ -656,7 +656,7 @@ function getArgumentsObject (tabId, sender, documentUrl) {
         site.brokenFeatures = site.brokenFeatures.concat(utils.getBrokenFeaturesAboutBlank(tab.url))
     }
 
-    if (tab?.site.isFeatureEnabled('trackingCookies3p')) {
+    if (tab?.site.isFeatureEnabled('trackingCookies3p') || tab?.site.isFeatureEnabled('trackingCookies1p')) {
         // determine the register domain of the sending tab
         const tabUrl = tab ? tab.url : sender.tab.url
```

Comment:
I'm interested when tab is blank here, do you know?

Should we change the about blank line on 656 to match?

Reply:
Good spot - given the check on like 633 `tab` cannot be falsy here.

I'll tidy up the optional chaining here as it seems to be used in many cases where it is not required.

Reply:
Can you remove the assignment of `tabUrl` as it's not needed also? Optionally move it to 640 also but make it:
```
const tabUrl = tab.url
```